test-util/config.ts

```diff
@@ -92,7 +92,7 @@ function isPortAvailable(port: number): boolean {
   }
 }
 
-function getRandomPort(min = 49152, max = 65534): number {
+export function getRandomPort(min = 49152, max = 65534): number {
```

Comment:
Given this is utilized elsewhere, want to extract it out of `test-util/config.ts` and into `test-util/port.ts`?